pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -100,6 +100,11 @@ spec:
         imagePullPolicy: {{ $.PullPolicy }}
         securityContext:
           runAsUser: 1
+        env:
+        - name: MY_POD_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP
```

Comment:
From my knowledge, k8s does not support this 1.5 years ago.  Are you sure downward api has supported podIP recenly?

Reply:
NVM, i remember we have use it already in injector

Reply:
I'm sure k8s support it all the time. And tested in v1.16